CHANGELOG.md

```diff
@@ -6,6 +6,11 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [10.34.1]
+### Fixed
+- Fix bug that could cause a failure in the persistence of network related data ([#20080](https://github.com/MetaMask/metamask-extension/pull/20080))
+- Fix  ([#20080](https://github.com/MetaMask/metamask-extension/pull/20080))
```

Comment:
Something went wrong with the changelog here. We should update this in the final changelog that we post.

Reply:
darn